sdcm/utils/gce_builder.py

```diff
@@ -0,0 +1,239 @@
+                file=sys.stderr,
+                flush=True,
+            )
+            print(f"Operation ID: {operation.name}", file=sys.stderr, flush=True)
+            raise operation.exception() or RuntimeError(operation.error_message)
+
+        if operation.warnings:
+            print(f"Warnings during {verbose_name}:\n", file=sys.stderr, flush=True)
+            for warning in operation.warnings:
+                print(f" - {warning.code}: {warning.message}", file=sys.stderr, flush=True)
+
+        return result
+
+    def _create_launch_template(self) -> compute_v1.InstanceTemplate:
+        """
+        Create a new instance template with the provided name and a specific
+        instance configuration.
+
+        Returns:
+            InstanceTemplate object that represents the new instance template.
+        """
+        # The template describes the size and source image of the boot disk
+        # to attach to the instance.
+        disk = compute_v1.AttachedDisk()
+        initialize_params = compute_v1.AttachedDiskInitializeParams()
+        initialize_params.source_image = (
+            self.runner.image.extra.get('selfLink')
+        )
+        initialize_params.disk_type = 'pd-standard'
+        initialize_params.disk_size_gb = 80
+        disk.initialize_params = initialize_params
+        disk.auto_delete = True
+        disk.boot = True
+
+        # The template connects the instance to the `default` network,
+        # without specifying a subnetwork.
+        network_interface = compute_v1.NetworkInterface()
+        network_interface.network = self.region.network.self_link
+
+        # The template lets the instance use an external IP address.
+        access_config = compute_v1.AccessConfig()
+        access_config.name = "External NAT"
+        access_config.type_ = "ONE_TO_ONE_NAT"
+        access_config.network_tier = "PREMIUM"
+        network_interface.access_configs = [access_config]
+
+        template = compute_v1.InstanceTemplate()
+        template.name = self.name
+        template.properties.disks = [disk]
+        template.properties.machine_type = "e2-medium"
+        template.properties.network_interfaces = [network_interface]
+
+        metadata = compute_v1.Metadata()
+        metadata.items += [{"key": "RunByUser", "value": "QA"}]
+        metadata.items += [{"key": "NodeType", "value": "builder"}]
+        metadata.items += [{"key": "keep", "value": "alive"}]
+
+        template.properties.metadata = metadata
+        template_client = compute_v1.InstanceTemplatesClient(credentials=self.credentials)
+        try:
+            operation = template_client.insert(
+                project=self.region.project, instance_template_resource=template
+            )
+
+            self._wait_for_extended_operation(operation, "instance template creation")
+        except google.api_core.exceptions.Conflict:
+            pass
```

Comment:
Maybe we want to print some useful message in this case?

Reply:
this mean it exists, if it's exist we don't recreate it, and move on to the next step.
not very critical thing to log, also we don't really have a log for those hydra actions.

Reply:
I didn't talk about `log`. I talked about `print` like you have above.
I understand the code, I wanted to highlight the fact that simple printed message that it already exists may be useful. It may be valueable to understand it, especially when we update the code fixing something.

Reply:
also probably would want to replace the `print` with `click.secho`... o.k. I'll change